pos_return_order/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
```

Comment:
Not sure to understand that point. copyright and author are something different. What do you suggest ?